spec/models/elastic_boosted_content_spec.rb

```diff
@@ -98,7 +106,7 @@
       end
 
       it 'should escape the entity but show the highlight' do
-        search = ElasticBoostedContent.search_for(q: 'carrot', affiliate_id: affiliate.id, language: affiliate.indexing_locale)
+        search = ElasticBoostedContent.search_for(q: 'carrots', affiliate_id: affiliate.id, language: affiliate.indexing_locale)
```

Comment:
Line is too long.  Try something like this below. It will also be easier to read.
```
search = ElasticBoostedContent.search_for(
  q: 'carrots',
  affiliate_id: affiliate.id, 
  language: affiliate.indexing_locale
)

```